hrms/hr/doctype/job_applicant/job_applicant.py

```diff
@@ -40,6 +40,11 @@ def validate(self):
 		if not self.applicant_name and self.email_id:
 			guess = self.email_id.split("@")[0]
 			self.applicant_name = " ".join([p.capitalize() for p in guess.split(".")])
+		
+		if self.job_title:
+			job_openings=frappe.get_doc("Job Openings",self.job_title)
+			if (job_openings.status=="Closed"):
+				frappe.throw("Cannot create job applicant against closed status job openings")
```

Comment:
This won't let me update an existing job applicant against a Closed job opening.
Move this validation to `before_insert` hook instead of `validate`